test/es_test_case.py

```diff
@@ -26,7 +26,7 @@ class ElasticsearchTestCase(DockerContainerTestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        es_endpoint = cls._create_container('docker.elastic.co/elasticsearch/elasticsearch:5.5.3',
+        es_endpoint = cls._create_container('docker.elastic.co/elasticsearch/elasticsearch:6.8.1',
```

Comment:
This is patch level 1. What patch level do we get on AWS?

Reply:
I am unsure as to what patch level is given by AWS, since I was unable to find that information from AWS. If you look at the [elasticsearch repo](https://github.com/elastic/elasticsearch), the branches they have available for listing do not show a patch level either.